packages/@aws-cdk/aws-apprunner-alpha/test/service.test.ts

```diff
@@ -1604,4 +1604,57 @@ test('create a service with a customer managed key)', () => {
       KmsKey: stack.resolve(key.keyArn),
     },
   });
-});
\ No newline at end of file
+});
+
+test('create a service with a Observability Configuration)', () => {
```

Comment:
```suggestion
test('create a service with a Observability Configuration', () => {
```
Similar for below test case.